site/posts/cartopy-tutorial.ipynb

```diff
@@ -0,0 +1,61 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "essential-strand",
+   "metadata": {},
+   "source": [
+    "The NCAR/UCAR virtual Python Tutorial Seminar Series continues with an introduction to the Python package `cartopy` on **Wednesday, April 28th at 1 PM Mountain**. This session will be led by Michaela Sizemore.\n",
+    "\n",
+    "The content for this tutorial is hosted on Michaelsa's [Cartopy Tutorial Github Repository](https://github.com/michaelavs/cartopy_tutorial)\n",
+    "\n",
+    "## Preparation\n",
+    "\n",
+    "- Check your conda install if this is the first tutorial in the series you are joining us for with `conda --version` and if necessary [install by following these instructions](https://docs.conda.io/en/latest/miniconda.html).\n",
+    "\n",
+    "<!-- TEASER_END -->\n",
+    "\n",
+    "- Navigate to Michaela's [Cartopy Tutorial Github Repository](https://github.com/michaelavs/cartopy_tutorial).\n",
+    "\n",
+    "- Clone this repository by clicking the green \"Code\" button and copying the SSH address under the word \"Clone\". This can be done by clicking the clipboard icon.\n",
+    "\n",
+    "- In your terminal, type `git clone PASTE` where you paste the SSH address you just copied (`git clone git@github.com:michaelavs/cartopy_tutorial.git`)\n",
+    "\n",
+    "- Create a Conda environment from the `conda_environment.yml` file with the terminal command `conda env create -f conda_environment.yml`. Make sure you are in the local `cartopy_tutorial` directory for this step.\n",
+    "\n",
+    "- Test your installation in the terminal with `conda activate cartopy_tutorial`. \n",
+    "\n",
+    "If you did not attend the live session, it will beneficial to watch the recording from the live tutorial on [Jupyter Notebooks](https://youtu.be/xSzXvwzFsDU) and on [Matplotlib](https://youtu.be/EiPRIdHQEmE) before the Cartopy session.\n",
```

Comment:
```suggestion
    "If you did not attend the live session, it will be beneficial to watch the recording from the live tutorial on [Jupyter Notebooks](https://youtu.be/xSzXvwzFsDU) and on [Matplotlib](https://youtu.be/EiPRIdHQEmE) before the Cartopy session.\n",
```